config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/studios", to: "studios#index"
+  get "/movies/:id", to: "movies#show"
+
+  post "/movies/:id", to: "actors#create"
```

Comment:
This is related to what I said about hte path in the other view file.... This path makes it seem like we're creating a movie, but then the `actors` controller is handling it.... which is also odd, because you're actually handling it in the `MovieActorsController`, and there is _no_ actors controller. If this wasn't quite working for you before, I think this may be the reason. Try double-checking this route and let me know what you find!